code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -24,7 +24,7 @@
 	if(!overmind)
 		create_overmind(new_overmind)
 	point_rate = new_rate
-	..(loc, h)
+	..(mapload, loc, h)
```

Comment:
Instance of where I am unsure with parent args

Reply:
Should just be . = ..(), and at the start of the proc